libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -202,10 +202,7 @@ class AP_InertialSensor : AP_AccelCal_Client
     uint8_t get_accel_filter_hz(void) const { return _accel_filter_cutoff; }
 
     // pass in a pointer to DataFlash for raw data logging
-    void set_dataflash(DataFlash_Class *dataflash) { _dataflash = dataflash; }
-
-    // enable/disable raw gyro/accel logging
-    void set_raw_logging(bool enable) { _log_raw_data = enable; }
+    void set_log_raw_log_bitmask_bit(uint32_t log_raw_log_bitmask_bit) { _log_raw_log_bitmask_bit = log_raw_log_bitmask_bit; }
```

Comment:
Will change as you suggest.  Yours didn't turn my brain inside out, so I'll rename.